cryptography/go/sdk/crypto-quickstart/app.go

```diff
@@ -0,0 +1,162 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"strings"
+
+	dapr "github.com/dapr/go-sdk/client"
+)
+
+const (
+	// Name of the crypto component to use
+	CryptoComponentName = "localstorage"
+	// Name of the RSA private key to use
+	RSAKeyName = "rsa-private-key.pem"
+	// Name of the symmetric (AES) key to use
+	SymmetricKeyName = "symmetric-key-256.b64"
+)
+
+func main() {
+	// Create a new Dapr SDK client
+	client, err := dapr.NewClient()
+	if err != nil {
+		log.Fatalf("Failed to initialize the Dapr client: %v", err)
+	}
+	defer client.Close()
+
+	// Step 1: encrypt a message, using the RSA key
+	encryptDecryptMessage(client)
+
+	// Step 2: encrypt a large file, using the AES key
+	encryptDecryptFile(client)
+}
+
+func encryptDecryptMessage(client dapr.Client) {
+	const message = "Ognuno sta solo sul cuor della terra\ntrafitto da un raggio di sole:\ned è subito sera."
```

Comment:
While I really love the personal style and flavor, it threw me off seeing Italian when I was trying to understand the encryption itself and if the STOUT looked right.  Would it be possibly ok to use English just so less of the WW audience needs to use Google?  I'm a believer that a quickstart has examples and domain knowledge everyone has, so you focus instead on the new concepts like Crypto itself.

Reply:
Yes, We should not use Italian here. English is the international standard, including in China.

Reply:
Yes, We should not use Italian here. English is the international standard, including in China.

Reply:
+1 to not having Italian here. English is the international that most countries accept and we need to keep a consistency in the examples. Please change to english